### PR TITLE
v2.0.2 release 

### PR DESCRIPTION
**Pull request by sfc-gh-tzhang**
Release v2.0.2, this will be used as our first GA release! This release is on branch https://github.com/snowflakedb/snowflake-ingest-java/tree/tzhang-si-ga since we only want to include selected changes on top of v2.0.1 to make sure it's a stable release, anything feature related will be released in v2.0.3 in the future.

### CODE REVIEW

**Review by sfc-gh-rcheng — APPROVED** [review]
lgtm, cla bot gate failing is ok since we are removing it in this pr

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <!-- Arifact name and version information -->
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-ingest-sdk</artifactId>
-  <version>2.0.1</version>
+  <version>2.0.2-SNAPSHOT</version>
```
Comment (sfc-gh-rcheng) [inline pom.xml:9]: Do we need your other [PR](https://github.com/snowflakedb/snowflake-ingest-java/pull/557) as well?

Comment (sfc-gh-xhuang) [reply pom.xml:9]: Also confused about the 2 PRs. 
Should this just be 2.0.2 ?

Snapshot release should be 2.0.3-SNAPSHOT

Comment (sfc-gh-tzhang) [reply pom.xml:9]: We're releasing on https://github.com/snowflakedb/snowflake-ingest-java/tree/tzhang-si-ga, so I'm porting every change that needs to be included manually one by one, the change comes from https://github.com/snowflakedb/snowflake-ingest-java/commit/e9103dd955061a12f0ce5414b0e138235d30a8b0, I will cherry-pick another which will override this to 2.0.2, see https://github.com/snowflakedb/snowflake-ingest-java/commit/79b4ea26e359046c4e347f71135e03879ff69923

**Review by sfc-gh-asen — APPROVED** [review]

**Pull request merged**